src/Common/Common.Api/Reports/ReportsClient.cs

```diff
@@ -25,24 +26,24 @@ public IEnumerable<Report> GetReports()
 
         public IEnumerable<Report> GetReportsAsAdmin(string filter = null, int? top = null, int? skip = null)
         {
-            return this.Client.Reports.GetReportsAsAdmin(filter: filter, top: top, skip: skip).Value?.Select(x => (Report)x);
+            return this.Client.Reports.GetReportsAsAdmin(Guid.Empty, filter: filter, top: top, skip: skip).Value?.Select(x => (Report)x);
```

Comment:
Is Guid.Empty intentional here as a passed parameter to the SDK? Is it possible that we could add support for whatever this parameter is?